src/serialization.c

```diff
@@ -135,6 +139,12 @@ raft_entry_t *RaftRedisCommandArraySerialize(const RaftRedisCommandArray *source
     p += n;
     sz -= n;
 
+    /* Encode cmd_flags */
+    n = encodeInteger('*', p, sz, source->cmd_flags);
```

Comment:
Actually, serialization here was using multi-bulk format with `\n` termination rather than `\r\n`. 
`*` indicates elem count in the list, so this is not correct but I see we already broke it. 
I guess we can revisit this later and fix it. Maybe we can combine some flags and make it more compact as well.

Reply:
Opened a ticket, https://redislabs.atlassian.net/browse/RR-232
We can revisit it later.